README.md

```diff
@@ -1,118 +1,254 @@
 
-In order to avoid update conflicts for frequently updated Digital IDs, Indexd uses a versioning system similar to that utilized in distributed version control systems. Within a particular Digital ID, this mechanism is referred to as the revision. For an update to take place, both the Digital ID and the revision must match that of the current Indexd document. When any update succeeds, a new revision is generated for the Indexd document. This prevents multiple, conflicting updates from occurring.
+It is then possible (via the API) to retrieve all versions for a given GUID. In addition, it is possible to ask for the _latest_ version of a GUID. See the [API documentation](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/Indexd/master/openapis/swagger.yaml) for more details.
 
-Digital IDs are intended to be publicly readable documents, and therefore contain no
-information other than resource locators. However, in order to prevent unauthorized
-editing of Digital IDs, each Digital ID keeps a list of authorization rules (authz).
-This authz string contains the rules of identities of users that have specific
-permissions for the associated Digital ID. For backward compatibility, the ACL list that
-was used for access control is still available, it would be used if authz is not present.
+But to reiterate, a given GUID will always point to the same data, even if there are later versions. The later versions will have _different_ GUIDs, though be connected through a common `baseid`. The Indexd API makes it possible to programmatically determine if newer versions of a given datum exist.
 
-The second layer of user defined aliases are introduced to add flexibility of supporting human readable identifiers and allow referencing existing identifiers that are created in other systems.
+### Access Control
+
+Indexd records (identified by GUIDs) are intended to be publicly readable documents, and therefore contain no information other than resource locators. However, in order to prevent unauthorized
+creation/updating/deleting of records, each record keeps a list of authorization rules (in an `authz` property).
+
+The `authz` property contains a list of abstract "resources" a user must have access to in order to have permission to update/delete the associated GUID. For backward compatibility, the ACL list that was used for access control is still available (the `acl` field).
+
+If Indexd is used with other Gen3 software, specifically the services related to Gen3 Auth ([Fence](https://github.com/uc-cdis/fence) and [Arborist](https://github.com/uc-cdis/arborist)), it enables a more useful and robust access control system that exposes various data access methods securely by utilizing the `authz` field in Indexd.
+
+The additional usage of the Gen3 Auth services will enable data access through signed URLs, with authorization checks based on the `authz` field in Indexd.
```

Comment:
there's not a public doc that I know of :( since the arborist / centralized auth stuff is pretty new. We probably _should_ have some cross-service docs though.... if you want to, you could make a story to do that